source/application/AMDatamanAppController.cpp

```diff
@@ -356,16 +356,16 @@ bool AMDatamanAppController::startupOnFirstTime()
 			return false;
 	}
 	else{
-		AMFirstTimeWizard ftw;
+		AMFirstTimeWizard *firstTimeWizard = new AMFirstTimeWizard(AMUserSettings::userBasedDataStorage);
 
```

Comment:
I don't see any parenting for the first time wizard. Will it need deleteLater() called on it at all the exit point to this block?